portfolio/src/main/webapp/index.html

```diff
@@ -5,15 +5,115 @@
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <!-- Center Image: Not working -->
+    <style type="text/css">
+    .centerImage
+    {
+    text-align:center;
+    display:block;
+    }
```

Comment:
Please fix formatting for readability. The cloud IDE you are using should have a an auto format (right click on the the text without highlighting anything and press format.